strapi/.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
Personally, I prefer 4 spaces ident

Reply:
I would agree for other languages, but 2 is often standard for JS / TS codebases, especially because of React component trees which can often nest rather deeply, e.g.
```tsx
const MyComponent = () => 
  <FooProvider>
    <Box>
      <Tabs>
        <Tab>
          <Card>
            <Typography>Title</Typography>
            ...
          </Card>
        </Tab>
      </Tabs>
    </Box>
  </FooProvider>
```